helix-tui/src/backend/termina.rs

```diff
@@ -238,6 +238,15 @@ impl TerminaBackend {
             log::debug!("terminfo could not be read, using default cursor reset escape sequence: {reset_cursor_command:?}");
         }
 
+        // Detect kitty multi-cursor support (available in kitty >= 0.43.0)
+        if matches!(
+            term_program().as_deref(),
+            Some("kitty") | Some("xterm-kitty")
```

Comment:
I think this detection should be changed to how its described in the kitty docs, otherwise other terminals supporting this feature will not be able to use it out of the box -> https://github.com/kovidgoyal/kitty/blob/master/docs/multiple-cursors-protocol.rst#querying-for-support